main/pipeline/pipeline.cc

```diff
@@ -231,21 +231,37 @@ unique_ptr<parser::Node> convertPrismToSorbet(pm_node_t *node, pm_parser_t *pars
             // Will only work for positive, 32-bit integers
             return make_unique<parser::Integer>(locOffset(loc, parser), std::to_string(intNode->value.value));
         }
+        case PM_OPTIONAL_PARAMETER_NODE: {
+            auto optionalParamNode = reinterpret_cast<pm_optional_parameter_node *>(node);
+            pm_location_t *loc = &optionalParamNode->base.location;
+            pm_location_t *nameLoc = &optionalParamNode->name_loc;
+
+            std::string_view name = prismConstantName(optionalParamNode->name, parser);
+            auto value = convertPrismToSorbet(optionalParamNode->value, parser, gs);
+
+            return make_unique<parser::Optarg>(locOffset(loc, parser), gs.enterNameUTF8(name),
+                                               locOffset(nameLoc, parser), std::move(value));
+        }
         case PM_PARAMETERS_NODE: {
             auto paramsNode = reinterpret_cast<pm_parameters_node *>(node);
             pm_location_t *loc = &paramsNode->base.location;
 
-            pm_node_list_t *requiredsList = &paramsNode->requireds;
-            auto requireds = absl::MakeSpan(requiredsList->nodes, requiredsList->size);
+            auto requireds = absl::MakeSpan((&paramsNode->requireds)->nodes, (&paramsNode->requireds)->size);
+            auto optionals = absl::MakeSpan((&paramsNode->optionals)->nodes, (&paramsNode->optionals)->size);
```

Comment:
`(&a->b)->c` can be simplified to just `a->b.c`

```suggestion
            auto requireds = absl::MakeSpan(paramsNode->requireds.nodes, paramsNode->requireds.size);
            auto optionals = absl::MakeSpan(paramsNode->optionals.nodes, paramsNode->optionals.size);
```

Derivation:

```c
(&a->b)->c`
(&(a->b))->c` // 1. Add `()` to clarify binding of `&`
(*&(a->b)).c` // 2. `x->y` is shorthand for `(*x).y`
(a->b).c      // 3. `*&x` cancels out to x
              //     "take the address of x, then find what's pointed to by it"
              //      ... it's just x.
a->b.c        // 4. remove unnecessary `()`
```